vpcinfo.go

```diff
@@ -163,3 +163,40 @@ type endpoints map[string]string
 func (e endpoints) String() string {
 	return fmt.Sprintf("list of %d resource(s)", len(e))
 }
+
+type multiPlatform []Platform
+
+func (p multiPlatform) String() string { return "multi" }
```

Comment:
This means the the string representation is basically useless since it will always be "multi" in the current implementation.

Instead of going for a fully generic `multiPlatform`, how about a simpler wrapper which only has one sub-platform so its `String` method can return the value of the sub-platform?